PWGEM/PhotonMeson/Core/PHOSPhotonCut.h

```diff
@@ -41,9 +41,10 @@ class PHOSPhotonCut : public TNamed
   static const char* mCutNames[static_cast<int>(PHOSPhotonCuts::kNCuts)];
 
   // Temporary function to check if track passes selection criteria. To be replaced by framework filters.
-  template <typename T>
-  bool IsSelected(T const& cluster) const
+  template <typename T, typename Cluster>
+  bool IsSelected(Cluster const& cluster) const
```

Comment:
same as for EMCal the `typename T` can be removed I think

Reply:
This is necessary to decide object type for track. In addition, IsSelected is called uniformly from PairUtilities.h. Therefore, IsSelected<typename>(cluster or v0) is necessary from all 3 cut classes.